src/paramak/parametric_reactors/submersion_reactor.py

```diff
@@ -60,7 +60,7 @@ def __init__(
         inboard_tf_leg_radial_thickness: float = 30,
         center_column_shield_radial_thickness: float = 30,
         inboard_blanket_radial_thickness: float = 80,
-        firstwall_radial_thickness: float = 20,
+        firstwall_radial_thickness: Union[float, list[float]] = 20,
```

Comment:
```suggestion
        firstwall_radial_thickness: Union[float, List[float]] = 20,
```

I would be tempted to use typing.Iterator instead of List here. But I think a capital L will fix the failing issue on the CI